tests/integration/session/test_login.py

```diff
@@ -365,6 +366,25 @@ def test_login_token_with_invalid_cir_instrument_id_results_in_500(self):
         # Then
         self.assertException()
 
+    @patch("app.routes.session.get_supplementary_data_v1")
+    @patch("app.routes.session._validate_supplementary_data_lists")
+    @patch(
+        "app.questionnaire.questionnaire_store_updater.QuestionnaireStoreUpdaterBase.set_supplementary_data",
+    )
+    def test_login_with_sds_schema_version_valid(
+        self, mock_get, mock_validate, mock_set
+    ):
+
+        self.launchSupplementaryDataSurvey(
+            schema_name="test_supplementary_data_with_sds_schema_version",
+            sds_dataset_id="203b2f9d-c500-8175-98db-86ffcfdccfa3",
+        )
+        # Then
```

Comment:
No "Given/When/Then" structure in this test, only "Then".

Reply:
Updated